platforms/hyperledger-besu/configuration/roles/create/prometheus/tasks/nested_loop.yaml

```diff
@@ -0,0 +1,17 @@
+- name: extracting data 
+  set_fact:
+    targets: []
+
+- name: validators
```

Comment:
The task name should be changed. Variable name `member` can create confusion, use `node` (which can be both member or validator). This doesn't check if Prometheus is enabled for node or not